oaknational/Oak-Web-Application
# feat: contact us page

## Description

- Creates `contact-us` page

## Issue(s)

Fixes #430 

## How to test

1. Go to {cloud link}
3. You should see the contact us page

**TODO** add brush card borders .... separate ticket

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/185941442-fc7f2558-a0fa-4802-876f-081e7ffb6e93.png)

![image](https://user-images.githubusercontent.com/12934669/185941598-f30b044c-4243-4d70-a8e8-7baaaaf63296.png)


## Checklist
